src/components/Calculator.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import './calculator.css';
+
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="calculator">
+        <div className="answer">
+          <span>0</span>
+        </div>
+        <div className="section1">
+          <button type="button" className="number">AC</button>
+          <button type="button" className="number">+/-</button>
+          <button type="button" className="number">%</button>
+          <button type="button" className="side-symbols">+</button>
```

Comment:
Make sure to have division operator which if you want you can use `/`. You have 2 `+` buttons at line 14 and line 16 make sure to match the design provided.

| Your Design | Provided design |
|-------------|---------------|
| <img width="319" alt="image" src="https://user-images.githubusercontent.com/34578631/182358438-f9cafcf9-58d0-47e5-8240-d1ee418fed9d.png"> | <img width="249" alt="image" src="https://user-images.githubusercontent.com/34578631/182358515-5b25112b-4f71-45b7-b227-1b698c807db3.png"> |

Reply:
@Mre55  fixed thanks for review